plugins/woocommerce/includes/class-wc-order-factory.php

```diff
@@ -22,7 +22,7 @@ class WC_Order_Factory {
 	 * Get order.
 	 *
 	 * @param  mixed $order_id (default: false) Order ID to get.
-	 * @return \WC_Order|bool
+	 * @return \WC_Order|false
```

Comment:
This update caused a havoc in phpstan-baseline because a lot of ignored errors were due to this, I've went down the call chain for `wc_get_order` and `get_order` and made sure it only return the order object or a refund or false, never true.